src/include/cell.i

```diff
@@ -67,6 +83,10 @@ __cell_pack_value_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_
         ++*pp;
 
         flags = 0;
+        if (durable_start_ts != WT_TS_NONE) {
```

Comment:
As I was discussing with Alex on a different point, how about storing the difference of `durable_start_ts - start_ts` as the durable value instead of the full value? This way the number of bytes required will be less.

Reply:
Great idea.  Done.